src/hooks/useBestTrade.ts

```diff
@@ -40,6 +40,7 @@ export function useBestTrade(
     useMemo(() => [amountSpecified, otherCurrency], [amountSpecified, otherCurrency]),
     useDebounceSwapQuoteFlag() === DebounceSwapQuoteVariant.Enabled ? DEBOUNCE_TIME_INCREASED : DEBOUNCE_TIME
   )
+  const inDebounce = (!debouncedAmount || !debouncedOtherCurrency) && Boolean(amountSpecified) && Boolean(otherCurrency)
```

Comment:
nit: I think `const inDebounce =
    (!debouncedAmount && Boolean(amountSpecified)) || (!debouncedOtherCurrency && Boolean(otherCurrency))` is slightly more readable to correlate that there has been an input but is currently being debounced

Reply:
another nit: I'd prefer `inDebounce` be placed directly above `isLoading`, as that's where it is used.